draft-ietf-ppm-dap.md

```diff
@@ -750,8 +758,7 @@ variant {
   /* Field exists regardless of variant */
   uint32 always_present;
   ExampleEnum type = number;
-  /* Only fields included in the `type == number`
-     variant is described */
+  /* Only fields included in the `type == number` variant are described */
```

Comment:
This line is just barely too long now

Reply:
Oh, whoops, I thought this needed to be <80 rather than <72.